tests/jballerina-integration-test/src/test/resources/websocket/src/wsservices/32_readyon_connect_retry.bal

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/io;
+
+@http:WebSocketServiceConfig {
```

Comment:
What is the purpose of this test case?

Reply:
You don't need a proxy to test readyOnConnect. You can use a server. Check 10_client_failure.bal